### PR TITLE
[IMP] project: improve ux

### PR DESCRIPTION
**Pull request by RaoufGhrissi**
**[IMP] project: improve ux**
**Before this commit:**
- When a project is created on SO confirmation,
the new project (assuming there is no project template
set on the product) is added with one stage with name=New
- When portal users drag & drop columns from the kanban view
of project.task, an access rights error is displayed.

**After this commit:**
- When a project is created on SO confirmation,
set the following stages for the new project
(assuming there is no project template set on the product):
to do - in progress - done (folded) - canceled (folded).
- The content of the task rating acknowledgment
page was centered vertically.
- Disable dragging and dropping project tasks for portal
user.

task-2929871

### CODE REVIEW

File: addons/sale_project/models/sale_order_line.py
Comment (auon-odoo) [inline addons/sale_project/models/sale_order_line.py:197]:
```suggestion
        project.type_ids = self.env['project.task.type'].create([{
            'name': name,
            'fold': fold,
            'sequence': sequence,
        } for name, fold, sequence in [
            (_('To Do'), True, 5),
            (_('In Progress'), True, 10),
            (_('Done'), False, 15),
            (_('Canceled'), False, 20),
        ]])
```
pif paf

**Review by auon-odoo — CHANGES_REQUESTED** [review]
Thank you :zap:

**Review by auon-odoo — APPROVED** [review]
Lgtm :ok_hand:

**Review by MissingNoShiny — COMMENTED** [review]
LGTM, aside from a minor comment that I'm not sure how to address (CSS :upside_down_face: )
Thanks for your work :slightly_smiling_face:

File: addons/rating/views/rating_templates.xml
Comment (RaoufGhrissi) [reply addons/rating/views/rating_templates.xml:6]: same for me, we may get another suggestion from the second review

Comment (xavierbol) [reply addons/rating/views/rating_templates.xml:6]: @MissingNoShiny @RaoufGhrissi don't forget the user can modify that page when website module is installed. By doing that changes, we could break the website style for that page.

I think we could just decrease the padding (or it was sufficient with classes you removed below).
Also, did you check in mobile view?

Finally, I don't think the user takes care of that detail. 😅

File: addons/sale_project/models/sale_order_line.py
Comment (xavierbol) [inline addons/sale_project/models/sale_order_line.py:155]: I will check with Alexandra if it is now really necessary to create those stages. 🙂

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I left 2 comments. 🙂

File: addons/sale_project/models/sale_order_line.py
Comment (RaoufGhrissi) [reply addons/sale_project/models/sale_order_line.py:155]: hey @xavierbol thank you for the review, have you discussed the centered div (previous comment) and this points with AJU ?

Comment (xavierbol) [reply addons/sale_project/models/sale_order_line.py:155]: Hello @RaoufGhrissi , yes you can see the discussion in the chatter of the task.

We will not do the centered div but we will just decrease the padding top (if needed)

For the stage, you can let as you did. 🙂

File: addons/rating/views/rating_templates.xml
Comment (xavierbol) [reply addons/rating/views/rating_templates.xml]: Hello @RaoufGhrissi ,

Can you remove that changes? Thanks in advance! 🙂

Comment (RaoufGhrissi) [reply addons/rating/views/rating_templates.xml]: Hello, maybe you need to refresh? i deleted it 4 days ago

File: addons/rating/static/src/scss/rating_templates.scss
Comment (xavierbol) [inline addons/rating/static/src/scss/rating_templates.scss:37]: It is not completely removed. 🧐

File: addons/rating/views/rating_templates.xml
Comment (xavierbol) [reply addons/rating/views/rating_templates.xml:6]: Maybe you have to push? I don't have the changes in your PR.

File: addons/rating/static/src/scss/rating_templates.scss
Comment (RaoufGhrissi) [reply addons/rating/static/src/scss/rating_templates.scss:37]: hello, i remember we showed the padding solution to Alexandra in the office and when zooming in, it wasn't really a good solution to center the text so if i'm not mistaken ( i can't find any trace as it was discussed offline ) we preferred to keep the actual solution. correct me if i'm wrong

Comment (xavierbol) [reply addons/rating/static/src/scss/rating_templates.scss:37]: See in the chatter of the task.

**Review by xavierbol — APPROVED** [review]
LGTM, thanks again for your work. 🙂 

robodoo r+

**Comment by xavierbol** [comment]
robodoo r+

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**
